cmd/copy.go

```diff
@@ -0,0 +1,103 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/unweave/cli/config"
+	"github.com/unweave/cli/ui"
+	"github.com/unweave/unweave/api/types"
+)
+
+var getSessionIDRegex = regexp.MustCompile(`^sess:([^/]+)`)
+
+func Copy(cmd *cobra.Command, args []string) error {
+	scpArgs := make([]string, 0, len(args))
+	var targetExec *types.Exec
+
+	for _, arg := range args {
+		argExecID, err := getExecIDFromCopyArgs(arg)
+		if err != nil {
+			return err
+		}
+
+		if argExecID != "" && targetExec != nil {
+			return fmt.Errorf("Copying between multiple sessions is not supported")
+		}
+
+		if argExecID != "" {
+			targetExec, err = getExecByNameOrID(cmd.Context(), argExecID)
+			if err != nil {
+				return fmt.Errorf("Could not find session by name or ID")
+			}
+		}
+
+		formattedArg, err := formatCopyArgToScpArgs(cmd.Context(), argExecID, targetExec, arg)
+		if err != nil {
+			return err
```

Comment:
This should not print the error to the user. This should be a `ui.Errof`. The user shouldn't see go errors